job_bundles/afterfx_render_one_task/template.yaml

```diff
@@ -73,15 +82,13 @@ steps:
   script:
     actions:
       onRun:
-        command: aerender
-        args:
-        - "-project"
-        - "{{Param.ProjectFile}}"
-        - "-comp"
-        - "{{Param.CompName}}"
-        - "-output"
-        - "{{Param.OutputDirectory}}"
-        - "-s"
-        - "{{Param.StartFrame}}"
-        - "-e"
-        - "{{Param.EndFrame}}"
+        command: bash
+        args: ['{{Task.File.Run}}']
+    embeddedFiles:
+    - name: Run
+      type: TEXT
+      data: |
+        mkdir -p '{{Param.OutputDirectory}}'
+
+        aerender -project '{{Param.ProjectFile}}' -comp '{{Param.CompName}}' \
+        -output '{{Param.OutputDirectory}}\{{Param.OutputFile}}' -s {{Param.StartFrame}} -e {{Param.EndFrame}}
```

Comment:
I find commands like this more readable if they're split a bit more systematically by the option. Here's one way you could do that:

```
        aerender -project '{{Param.ProjectFile}}' \
            -comp '{{Param.CompName}}' \
            -output '{{Param.OutputDirectory}}\{{Param.OutputFile}}' \
            -s {{Param.StartFrame}} \
            -e {{Param.EndFrame}}
```

Reply:
Thank you for the suggestion. Updated the template.